data/lib/debugging/dump.lua

```diff
@@ -0,0 +1,51 @@
+-- recursive dump function
+function dumpLevel(input, level)
+   local indent = ''
+   for i = 1, level do
+      indent = indent .. '    '
```

Comment:
Isn't \t better here to print out a tab?

Reply:
`\t` often defaults to 8 spaces which I dislike. It also looks aweful in-game when sending as a text message. 

First pdump is `\t`, second is just 4 blank spaces:
![image](https://user-images.githubusercontent.com/1552144/76712544-3d17c500-671a-11ea-9aac-4d28d5df361d.png)

Reply:
True, looks better :)